lib/Glimmer.pm

```diff
@@ -0,0 +1,494 @@
+# use strict;
+# use Data::Dumper;
+
+# my $out = Glimmer::call_genes_with_glimmer(file_with_path, { verbose => 1 });
+
+# print Dumper($out);
+
+package Glimmer;
+use strict;
```

Comment:
All perl files (scripts, modules, etc.) should have the `strict` and `warnings` pragmas turned on -- i.e. they should have

```
use strict;
use warnings;
```

at the top.